src/main/java/com/hyperwallet/clientsdk/model/HyperwalletUser.java

```diff
@@ -37,8 +46,8 @@
     private String businessName;
     private String businessOperatingName;
     private String businessRegistrationId;
-    private String businessRegistrationStateProvince;
     private String businessRegistrationCountry;
+    private String businessRegistrationStateProvince;
     private BusinessContactRole businessContactRole;
```

Comment:
It looks like the Businesscontactrole field is no longer there in V4. Could you please check this - https://confluence.site1.hyperwallet.local/pages/viewpage.action?pageId=74715055

Reply:
This needs clarification as there is documentation in the same page about a condition that needs businessContactRole.

**letterOfAuthorizationStatus**

Letter of authorization is required only for a business in an EEA country and only if the business contact of the business in not listed as one of the company directors